repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -610,3 +583,16 @@ export const clearDates = (): void => {
     getElementById('end-month-input').clear();
     getElementById('end-year-input').clear();
 };
+
+export const completeMultiServicePages = (): void => {
+    
+    cy.get('[class=govuk-checkboxes__item]').each((checkbox) => {
+        cy.wrap(checkbox).click();
+    });
+    
+    getElementById('operator-1').click();
+    cy.get('[class=govuk-checkboxes__item]').each((checkbox) => {
+        cy.wrap(checkbox).click();
+    });
+    continueButtonClick();
```

Comment:
We don't have anything to verify it is finishing like isFinished() in other product journeys. We could probably also check if that product is in the multi operator product page. As an extension to that even check if it has the right details (but that is optional).

Reply:
resolved we are being consistent as long as it reaches the last page it is fine